glide.yaml

```diff
@@ -42,3 +42,6 @@ import:
   version: 89742aefa4b206dcf400792f3bd35b542998eb3b
 - package: github.com/blang/semver
   version: b38d23b8782a487059e8fc8773e9a5b228a77cb6
+
+- package: github.com/imdario/mergo
+  version: 6633656539c1639d9d78127b7d47c622b5d7b6dc
```

Comment:
not what I see in kube.  What's up? https://github.com/kubernetes/kubernetes/blob/master/Godeps/Godeps.json#L2336